rest-api-spec/src/main/resources/rest-api-spec/test/indices.create/10_basic.yml

```diff
@@ -31,6 +31,16 @@
   - match: { test_index.settings.index.number_of_replicas: "0"}
 
 ---
+"Create index":
+
```

Comment:
This will need a skip because it will fail the backwards compatibility tests without it. Typically what we would do is skip through to 5.99.99, then we backport this and set the skip to 5.5.99, then we change the skip in master to 5.5.99. (We handle the backporting, so you can simply set the skip to 5.99.99 in this PR.)